### DISCUSSION TITLE
graphql/scalars/docs/usage

### DISCUSSION THREAD

**Discussion by giscus** [Urigo/graphql-scalars#1837] [Docs Discussions]
# graphql/scalars/docs/usage

GraphQL Scalars Documentation

https://the-guild.dev/graphql/scalars/docs/usage

**Comment by isaackhlam** [comment]
Does it a typo?
> Notw that you can also import ready-to-use type definitions for scalars like below:

**Reply by ardatan** [reply]
Fixed! Thanks for reporting it!
